hummingbird/admin.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from hummingbird.models import UserProfile, UserDevice
+
+class UserProfileAdmin(admin.ModelAdmin):
+	list_display = ('name','last_played','song','length',)
+
+admin.site.register(UserProfile,UserProfileAdmin)
```

Comment:
nit: space between UserProfile and UserProfileAdmin
ie. `admin.site.register(UserProfile, UserProfileAdmin)`